bedrock/mozorg/templates/mozorg/about/policy/transparency/includes/past-reports.html

```diff
@@ -9,6 +9,9 @@
     <nav>
       <h3>{{ _('Previous Reports') }}</h3>
       <ul>
+        {% if switch('transparency-report') %}
```

Comment:
It might be good to add a date to the switch name, just for context. Though really we should clean up the switch right after it goes live anyway so it probably doesn't matter. Up to you.

And I also think we're now supposed to favor underscores instead of dashes in switch names since that's what www-config uses, but I don't think it's documented anywhere. The config will match dashes and since this should be short lived it's still up to you if you want to change it again, no biggie.